jbpm-test-coverage/src/test/java/org/jbpm/test/functional/jobexec/LogCleanupCommandTest.java

```diff
@@ -280,26 +280,27 @@ public void deleteAllLogsOlderThanTomorrowDifferentProcess() throws Exception {
         Assertions.assertThat(getNodeInstanceLogSize(HELLO_WORLD_ID)).isPositive();
     }
 
-    @Test(timeout = 10000)
```

Comment:
I wouldn't remove the timeout. Since it's using a countDownLatch, this test in case of error might be hanging forever waiting for the latch to complete. I would keep this to interrupt the test in that scenario

Reply:
yes, you are right, restored